Gems/LyShine/Code/Source/Sprite.cpp

```diff
@@ -699,17 +707,8 @@ CSprite* CSprite::CreateSprite(const AZStd::string& renderTargetName)
     // create Sprite object
     CSprite* sprite = new CSprite;
```

Comment:
No current plan to remove CSprite. Much of its content has legacy concepts, but it does contain some extra features such as 9-slice sprite and sprite sheet formats. It would benefit from a refactor though.

Reply:
Okay good to know, thanks for the info @michabr. Might be worth in future renaming to lose the old Hungarian notation `C` prefix if and having it use new AZ types etc...